### ISSUE TITLE
Crash when Aseprite attempts to navigate to the most recent folder after it has been deleted

### ISSUE THREAD

**Issue by ShortRoundDev** [aseprite/aseprite#2158]
After exporting a sprite to a folder with a subfolder (i.e: `Project/TestFolder/SubFolder/MySprite.png`), if `TestFolder` is deleted, re-opening the Export dialog can cause Aseprite to crash.

This one seems to me to be an edge case, but I did run into it naturally.

Steps to reproduce

1. Open Aseprite
2. Create a sprite
3. In some folder, e.g, `Project`, create a new folder (call it, say, `TestFolder`)
4. Inside `TestFolder`, create a subfolder (call it, say, `Subfolder`).
5. Export your new sprite to `Subfolder`
6. In Explorer, navigate to `Project` and deleted `TestFolder`
7. Open the dialog to export the sprite again, and you will be shown a blank file explorer dialog which appears to be on "Desktop", but nothing exists in this explorer window
8. Pressing "Back" will cause an error dialog to appear, and then navigate you to the now-non-existent `Subfolder` directory
9. Pressing the "Directory Up" button will cause the same error to appear
10. Pressing Ok on this error causes Aseprite to crash

In case this is hard to follow, I recorded an example which I will include a link to here

### Aseprite and System version

* Aseprite v1.2.5-x64, Steam Version
* System: Windows Build 1803

https://i.imgur.com/VbjBqVk.mp4

**Comment by dacap** [comment]
I think the crash should be fixed since v1.2.16 (change introduced in b5b450afba909df3065111dfb366243298bfff12), anyway the step where the "Desktop" folder appears empty is still reproducible (but there is no –or should not be a– crash)

**Comment by iamOgunyinka** [comment]
I can confirm it has.

**Comment by martincapello** [comment]
I tested this again, so closing because it was fixed a long time ago.
